README.md

```diff
@@ -95,7 +95,23 @@ import com.google.common.collect.*;
 
 import static com.faunadb.client.query.Language.*;
 
+class Person {
+  @FaunaField
+  private final String firstName;
+  @FaunaField
+  private final String lastName;
+
+  @FaunaConstructor
+  public Person(@FaunaField("firstName") String firstName, @FaunaField("lastName") String lastName) {
+    this.firstName = firstName;
+    this.lastName = lastName;
+  }
+}
+
 public class Main {
+  private static final Field<Value> DATA_FIELD = Field.at("data");
```

Comment:
Just to save some space on the page. I'd also inline the annotations above with their field definitions. This README is already huge!